dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -205,6 +206,8 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
                         wil::unique_cotaskmem_string processName;
                         THROW_IF_FAILED(GetCurrentProcessPath(processName));
+                        std::wstring appId{ RetrieveAppId() };
```

Comment:
If you see above - there is a parameter called unpackagedAppUserModelId which you get it from the LRP (note you also register this Identifier in the wpn platform by calling registerfulltrustapplication)? Should you be using the same appId and register it in the registry?

Reply:
If you call ToastNotificationManager::RegisterActivator for the retrieveAppId after getting the channel then it will create a new registration in the wpnplatform for the appId. Is that intended? One for toasts and one for raw for the same processname?

Reply:
Please infer to the design doc from @danielayala94 which runs through the mapping flow. In essence, we decided that the ToastREgistrationId and PushREgistrationId are different entities. The reason for this is simple, we don't want to bring up LRP(Long Running Process) for toast scenarios for perf reasons unless we also have a dependency on Push.